u2-routing-and-controllers/features/tempConreoller.js

```diff
@@ -0,0 +1,20 @@
+const getTemp = (req, res) => {
+  const unit1 = req.params.unit1.toUpperCase()
+  const unit2 = req.params.unit2.toUpperCase()
+  const temp1 = req.params.temp1
+
+  let temp2
+  if (unit1 === unit2) {
+    temp2 = temp1
+  } else if (unit1 === 'C') {
+    temp2 = temp1 * 9 / 5 + 32
+  } else if (unit1 === 'F') {
+    temp2 = (temp1 - 32) * 5 / 9
+  }
+
+  res.send(`${parseFloat(temp1).toFixed(1)} ${unit1} = ${parseFloat(temp2).toFixed(1)} ${unit2}`)
```

Comment:
จะดีกว่าที่จะแยกวิเคราะห์ตัวเลขทันที นี่เป็นความผิดของฉัน ฉันคิดว่าฉันไม่ได้ทดสอบวิธีแก้ปัญหาอย่างถูกต้อง ฉันแนะนำสิ่งนี้ / It's better to parse the number immediately. This is my fault -- I think I didn't test my solution properly. I recommend:
```js
const t1 = parseFloat(req.params.t1)
...
res.send(`${t1.toFixed(1)} ${u1} = ${t2.toFixed(1)} ${u2}`)
```

`t2` เป็น float (floating-point number) โดยอัตโนมัติเนื่องจากวิธีคำนวณ
`t2` is automatically a float (floating-point number) due to how it's calculated